src/main/kotlin/io/github/moulberry/notenoughupdates/miscfeatures/OldSkyBlockMenu.kt

```diff
@@ -147,29 +146,25 @@ object OldSkyBlockMenu {
         ),
         ;
 
-        fun buildItem(): ItemStack {
-            if (cachedItem != null) return cachedItem!!
-
+        val item: ItemStack by lazy {
             val lore = mutableListOf<String>()
             for (line in displayDescription) {
                 lore.add("§7$line")
             }
             lore.add("")
             lore.add("§eClick to execute /$command")
             val array = lore.toTypedArray()
-
             val name = "§a$displayName"
-            val result = when (itemData) {
+            when (itemData) {
                 is NormalItemData -> Utils.createItemStackArray(itemData.displayIcon, name, array)
                 is SkullItemData -> Utils.createSkull(name, itemData.uuid, itemData.value, array)
```

Comment:
you can make `ItemData` a `sealed interface` which allows you to drop the extra else branch

Reply:
Sealed is really funny